installer/build/build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this file is responsible for parsing cli args and spinning up a build container
+# wraps bootable/build-main.sh in a docker container
+
+DEBUG=${DEBUG:-}
+set -e -o pipefail +h && [ -n "$DEBUG" ] && set -x
+ROOT_DIR="$GOPATH/src/sigs.k8s.io/cluster-api-provider-vsphere"
+ROOT_WORK_DIR="/go/src/sigs.k8s.io/cluster-api-provider-vsphere"
+
+ROOT_INSTALLER_DIR="${ROOT_DIR}/installer"
+ROOT_INSTALLER_WORK_DIR="${ROOT_WORK_DIR}/installer"
+
+# TAG=${DRONE_TAG:-$(git describe --abbrev=0 --tags)} # e.g. `v0.9.0`
+TAG="v0.0.1"
```

Comment:
Is hard-coding the version intentional? (The above reference to `DRONE_TAG` should be removed, at least.)

Reply:
Yes, it is intentional, I will be adding a comment to better explain it in code